MiniApp/Classes/JavascriptBridge/MiniAppScriptMessageHandler.swift

```diff
@@ -83,6 +88,22 @@ internal class MiniAppScriptMessageHandler: NSObject, WKScriptMessageHandler {
         }
     }
 
+    func getCurrentPosition(callbackId: String) {
+
+        self.executeJavaScriptCallback(responseStatus: .onSuccess, messageId: callbackId, response: getLocationInfo())
+    }
+
+    func getLocationInfo() -> String {
+        return  "{\"coords\":{\"latitude\":\(locationManager?.location?.coordinate.latitude ?? 0)" +
+        ",\"longitude\":\(locationManager?.location?.coordinate.longitude ?? 0)" +
+        ", \"altitude\":\(locationManager?.location?.altitude ?? 0)" +
+        ", \"altitudeAccuracy\":\(locationManager?.location?.verticalAccuracy ?? 0)" +
+        ", \"accuracy\":\(locationManager?.location?.horizontalAccuracy ?? 0)" +
+        ", \"speed\":\(locationManager?.location?.speed ?? 0)" +
+        ", \"heading\":\(locationManager?.location?.course ?? 0)" +
```

Comment:
It looks like `altitude`, `altitudeAccuracy`, `speed`, and `heading` are nullable (see https://developer.mozilla.org/en-US/docs/Web/API/GeolocationCoordinates). Is it possible to change the fallback value to `null` instead of `0`?

Reply:
Done 👍